spec/fixtures/forms/2021_2022.json

```diff
@@ -1,638 +1,708 @@
 {
-    "form_type": "lettings",
```

Comment:
Why is the Lettings Log JSON changing? If this is just formatting might be worth splitting that out of this PR?

Reply:
I'll clean up the git history in a moment, then you can review commit by commit.

This changed because two reasons, formatting and I have added inferred answers to one of the questions

Reply:
I think I'd recommend moving the formatting changes to their own PR still. Purely because it's a lot of lines, hard to review thoroughly and so far considered config so not well tested (this should ideally change).

Reply:
please review this one https://github.com/communitiesuk/submit-social-housing-lettings-and-sales-data/pull/923